static/script.js

```diff
@@ -1,4 +1,14 @@
+document.querySelectorAll(".emoji-button").forEach(function (button) {
+  button.addEventListener("webkitAnimationEnd", function() {
+    // reset animation
+    button.style.animationName = "";
+  });
+});
+
 function post_feedback(fb) {
+  var button = document.querySelector(".emoji-button." + fb);
```

Comment:
Well, my intent was to support [only modern Chrome/Chromium browsers](https://github.com/tum-rt/emoji-feedback/blob/e35f74114d2631ae503bbe597ba46af58ff2585c/README.md#disclaimer).
That's why I kind of frowned at your standard/Mozilla/WebKit JS cases, but hey.

I dunno, it actually feels kind of liberating to support only one (modern) browser and 2 platforms...